freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/model/FmsGroup.java

```diff
@@ -6,8 +6,9 @@
 
     private String name;
 
-    public void setName(String name) {
+    public FmsGroup setName(String name) {
```

Comment:
usually set methods don't return, I think builder pattern use the `with` naming convention